dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -1232,8 +1233,8 @@ BlockInputStreams DeltaMergeStore::read(const Context & db_context,
             filter,
             max_version,
             expected_block_size,
-            false,
-            db_settings.dt_raw_filter_range,
+            is_raw_read,
+            true,
```

Comment:
```suggestion
            /*do_delete_mark_filter_for_raw_=*/true, // always ensure not returning the "delete mark" rows
```

Reply:
do_delete_mark_filter_for_raw means we do not do del_mark = 1 filter, but we may return the columns includes del column.